src/coreComponents/constitutive/unitTests/TestFluid.hpp

```diff
@@ -67,6 +67,7 @@ class TestFluid
     createArray( testFluid->criticalTemperature, components, Fluid::Tc, Fluid::data );
     createArray( testFluid->acentricFactor, components, Fluid::Ac, Fluid::data );
     createArray( testFluid->molecularWeight, components, Fluid::Mw, Fluid::data );
+    createArray( testFluid->volumeShift, std::vector< real64 >( NC, 0.0 ) );
```

Comment:
Can we not optionally refactor that for data to be append from `Fluid::data` as the others even though it is only zeros ?